recipes/hangar/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "hangar" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85bbb1e8fe0cac56ca86257283a96472bd8410443d1b819d0e6b67f0a0d63214
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  entry_points:
+    - hangar = hangar.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

As this isn't `noarch` you can't use this way of containing the Python version. Instead you have one build per Python version per platform.
```suggestion
    - python
```

---

fixed!